images/SMRT.svg

```diff
@@ -0,0 +1,1048 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
File size should be <15kB - can you try to improve your logo?

Reply:
My apologies our graphics guy is out and its not my strong suite , to say the least. I've updated the svg to a newer, smaller & smaller sized image. Fingers crossed.